examples/restclient-pact/README.md

```diff
@@ -0,0 +1,7 @@
+# Quarkus RestClient
+
+Showcase demonstrating how to test a RestClient with PACT
+
+As the reactive RestClient should be a drop in replacement for the _normal_ RestClient we use in our
+showcase only the reactive RestClient
```

Comment:
I understand this sentence only in parts ... maybe everything from `only ..` is a mistake?